### PR TITLE
Add RubyMotion

### PR DESCRIPTION
**Pull request by mipmip**
**https://github.com/motion-open-source/awesome-rubymotion**

**RubyMotion let's you write cross-platform apps for iOS, Android, tvOS, watchOS and OS X in Ruby. This awesome list has been around for 2 years now, and has 45 stars. This list tries to give a complete overview of resources about RubyMotion** 

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
-  The entry in the Awesome list should:
	- [x] Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- [x] Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- [x] It's the result of hard work and the best I could possibly produce.
	- [x] Non-generated Markdown file in a GitHub repo.
	- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- [x] Not a duplicate.
	- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
	- [x] Includes a project logo/illustration whenever possible.
		- [x] Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- [x] The image should link to the project website or any relevant website.
		- [x] The image should be high-DPI. Set it to maximum half the width of the original image.
	- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- [x] Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- Has a Table of Contents section.
		- [x] Should be named `Contents`, not `Table of Contents`.
		- [x] Should be the first section in the list.
	- [x] Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- [x] That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [x] If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- [x] The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- [x] Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- [x] Drop all the `A` / `An` prefixes in the descriptions.
		- [x] Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.

**Comment by mipmip** [comment]
To comply, I updated the following:
	
- replace all os x with macOS
- remove a/an prefixes
- add logo, add licence
- more descriptive introduction at top of README

**Comment by sindresorhus** [comment]
Some more feedback:

---

Note this about the header:

> The image should be high-DPI. Set it to maximum half the width of the original image.

There's also no need for the text list title, as it's just duplicating the header image. The Awesome badge can be placed below the intro.

---

`RubyMotion` in the intro should link to https://en.wikipedia.org/wiki/RubyMotion

---

> Each link description starts with an uppercase character and ends with a period.

Not done on all items.

---

I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community

File: readme.md [hunk 1]
```diff
@@ -82,6 +82,7 @@
 - [Heroku](https://github.com/ianstormtaylor/awesome-heroku)
 - [Raspberry Pi](https://github.com/thibmaek/awesome-raspberry-pi) - Credit card-sized computer aimed at teaching kids programming, but capable of a lot more.
 - [Qt](https://github.com/JesseTG/awesome-qt) - Cross-platform GUI app framework.
+- [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, MacOS, tvOS and watchOS in Ruby
```
Comment (sindresorhus) [inline readme.md]: Missing a dot, and it's `macOS`, not `MacOS`.

**Comment by mipmip** [comment]
Thanks for your comments. 

I updated the README and de pull request.

**Comment by sindresorhus** [comment]
I don't see this being fixed:

> Note this about the header:
> The image should be high-DPI. Set it to maximum half the width of the original image.

**Comment by mipmip** [comment]
Oops.. it's done now.

**Comment by sindresorhus** [comment]
Added :)

**Comment by mipmip** [comment]
Thanks

**Pull request merged**